include/Core/Users/DeleteUsersPage.php

```diff
@@ -21,6 +21,40 @@ class DeleteUsersPage extends BaseDeletePage {
 	 * @since 5.5
 	 */
 	protected function initialize() {
+		if ( is_multisite() ) {
+			$this->multisite_init();
+		} else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method initialize uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/app/BulkWP/bulk-delete/pullRequest?prid=4259873)